scripts/superdesk-authoring/authoring/controllers/ChangeImageController.js

```diff
@@ -1,5 +1,7 @@
-ChangeImageController.$inject = ['$scope', 'gettext', 'notify', 'modal', '$q', 'lodash', 'api', '$rootScope', 'config'];
-export function ChangeImageController($scope, gettext, notify, modal, $q, _, api, $rootScope, config) {
+ChangeImageController.$inject = ['$scope', 'gettext', 'notify', 'modal', '$q', 'lodash', 'api', '$rootScope', 'config',
+                                 'authoringWorkspace', 'archiveService'];
+export function ChangeImageController($scope, gettext, notify, modal, $q, _, api, $rootScope, config, authoringWorkspace,
+                                      archiveService) {
```

Comment:
Please do, I think best would be 4 spaces indentation for both of those lines, unless anyone has objections. I think that would be most common approach and style.

Re. tests, keep in mind that some are failing because of https://github.com/superdesk/superdesk-client-core/pull/935 - which is about to fix them.